### PR TITLE
Tide history: consistent poolKey to repo branch conversion

### PR DESCRIPTION
**Pull request by chaodaiG**
They were handled in different code path before

### CODE REVIEW

File: prow/cmd/deck/static/tide-history/tide-history.ts [hunk 1]
```diff
@@ -182,12 +188,10 @@ function redraw(): void {
   const hist: {[key: string]: Record[]} = typeof tideHistory !== 'undefined' ? tideHistory.History : {};
   const poolKeys = Object.keys(hist);
   for (const poolKey of poolKeys) {
-    const match = RegExp('(.*?):(.*)').exec(poolKey);
```
Comment (listx) [inline prow/cmd/deck/static/tide-history/tide-history.ts:185]: This regex is not exactly the same as `RegExp('(((https|http)://[^/]*/)?.*?)` but I guess functionally they are still equivalent? Do we have tests that ensure we're not breaking anything here?

Comment (chaodaiG) [reply prow/cmd/deck/static/tide-history/tide-history.ts:185]: I have done runlocal and at least it works for github

**Pull request merged**